libkbfs/block_retrieval_queue.go

```diff
@@ -6,19 +6,19 @@ package libkbfs
 
 import (
 	"container/heap"
+	"io"
 	"sync"
 
 	"golang.org/x/net/context"
 )
 
 const (
-	defaultBlockRetrievalWorkerQueueSize int = 100
+	defaultBlockRetrievalWorkerQueueSize int = 20
```

Comment:
Why reduce this?  I think we have a default parallelism of 100 in `folderBranchOps` that this would stifle.